docs/content/examples/tutorials/adguard.md

```diff
@@ -1,9 +1,91 @@
+# AdGuard Home with WG Easy and Watchtower (Docker Compose / Host Mode)
+
+## 📌 Description
+This setup runs **AdGuard Home**, **WG Easy** (WireGuard management UI) and **Watchtower** for automatic updates — all in **Docker host network mode**.
+
 ---
-title: AdGuard Home
----
 
-It seems like the Docs on how to setup AdGuard Home are not available yet.
+## 📂 Prepare Local Folders
+
+Before starting, create the required directories:
+
+```bash
+sudo mkdir -p /home/admin/docker/wireguard
+sudo mkdir -p /home/admin/docker/adguard/work
+sudo mkdir -p /home/admin/docker/adguard/conf
+mkdir -p /home/admin/docker/watchtower
+
+Save as docker-compose.yml:
+
+services:
+  wg-easy:
+    image: ghcr.io/wg-easy/wg-easy:15
+    container_name: wg-easy
+    environment:
+      - WG_HOST=your_Domain   # External domain / Router must forward port
+      - WG_PORT=51820         # wg-easy listens on 51820/udp
+      - WG_DEFAULT_DNS=10.8.0.1   # AdGuard runs on the same host
+      - WG_ALLOWED_IPS=0.0.0.0/0,::/0
+      - INSECURE=true
+      - DISABLE_IPV6=false    # Set to true to disable IPv6
```

Comment:
the `WG_*` vars do not even exist. and the other not recommended env vars should not be here, as they are already in the docs